### ISSUE TITLE
Ion-slide , first image problem

### ISSUE THREAD

**Issue by kazmierczak461** [driftyco/ionic#9394]
Ionic version 2 rc3

I made a thumbnail list with images. After click on the image, image is opening in ion-slide. 
After loading the ion-slide, each time shown is the first picture from ng-repeat for about a second (just quick blink), then after that the right image.

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! Would you be able to provide a [plunker](http://plnkr.co/edit/GJte2b?p=preview) that demonstrates this issue? Thanks for using Ionic!

**Comment by AleDurand** [comment]
Hello, I am using the same version and I was able to reproduce the issue.

Here an example using numbers instead of images: [http://plnkr.co/edit/WrwhdG?p=preview](http://plnkr.co/edit/WrwhdG?p=preview)

I hope it helps.

Regards.
Alejandro.

**Comment by kazmierczak461** [comment]
Yes exactly, this is it! :) . I hide this bug in my project by fadein in css, but still would be good to fix it.

**Comment by jgw96** [comment]
Thanks for the plunker @AleDurand !
